src/casp/bq_scripts/extract_minibatch_to_anndata.py

```diff
@@ -199,25 +207,36 @@ def extract_minibatch_to_anndata(
         feature_ids.append(feature.original_feature_id)
         cas_feature_index_to_col_num[feature.cas_feature_index] = col_num
 
+    # Getting rid of table aliases in `obs_columns_to_include`
+    _obs_columns_to_include = list(map(lambda x: x.split(".")[-1], obs_columns_to_include))
```

Comment:
This is potentially unsafe in case the obs column contains `.`. Let's throw an exception in case there are multiple periods.